### PR TITLE
UIUX-57 Return response duration

### PR DESCRIPTION
**Pull request by EngineersBox**
# Changelog

* Added response duration field as `duration`
* Cleaned up formatting and comments

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -96,6 +104,7 @@ def base_search(query: str, search_method: Callable[[Elasticsearch, str, str], N
         orderedLoad = lambda x: json.loads(x, object_pairs_hook=OrderedDict)
         result_value = list(map(orderedLoad, rd.execute_command("LRANGE", query, 0, -1)))
         return jsonify(result={
+            "id"
```
Comment (jayjayjayjayjayjay) [inline app.py]: If we already get the id in the response, is this addition would be unnecessary?

Comment (EngineersBox) [reply app.py]: Yeah this was a mistake, seems like I forgot to remove it.

**Comment by EngineersBox** [comment]
@jayjayjayjayjayjay Good for re-review, i've removed that unused `id` field

**Comment by jayjayjayjayjayjay** [comment]
LGTM

**Review by jayjayjayjayjayjay — APPROVED** [review]
LGTM

**Pull request merged**